js/views/search/Search.js

```diff
@@ -0,0 +1,229 @@
+import baseVw from '../baseVw';
+import loadTemplate from '../../utils/loadTemplate';
+import app from '../../app';
+import $ from 'jquery';
+import Dialog from '../modals/Dialog';
+import Results from './Results';
+import ResultsCol from '../../collections/Results';
+import SettingsModal from '../modals/Settings/Settings';
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    const term = options.term;
+    const testForURL = /^((http|https|ob):\/\/)/;
+
+    if (term && testForURL.test(term)) {
+      // if a search URL was passed in, reconstruct the url and parse the data
+      const searchURL = new URL(`${term}${options.query ? `?${options.query}` : ''}`);
+      const params = searchURL.searchParams;
+      this.sProvider = `${searchURL.origin}${searchURL.pathname}`;
+      this.serverPage = params.get('p') || 0;
+      this.pageSize = params.get('ps') || 12;
+      this.term = params.get('q') || '';
+      this.callSearchProvider(searchURL);
+    } else {
+      this.sProvider = app.localSettings.get('searchProvider');
+      this.serverPage = options.serverPage || 0;
+      this.pageSize = options.pageSize || 12;
+      // if the term was not a url, process the term before calling the search provider
+      this.processTerm(term);
+    }
+
+    this.usingDefault = this.sProvider === app.localSettings.get('searchProvider');
+
+    // if not using a passed in URL, update the default provider if it changes
+    this.listenTo(app.localSettings, 'change:searchProvider', (_, provider) => {
```

Comment:
Sure. This was just a copy paste from something else that used an underscore there.